pipeline/pipelines/code_generation_pipeline.py

```diff
@@ -271,3 +271,20 @@ def do_build_flow(self, **kwargs):
 
     def validate_kwargs(self, **kwargs):
         _validate_codegen_kwargs(_VGEN_REQUIRED, **kwargs)
+
+
+class VkitConfigPipeline(pipeline_base.PipelineBase):
```

Comment:
I would suggest putting this before the other pipelines, because in the process of building a client, this would be the first thing that is used.